cmd/fetchd/cmd/genesis-asi-upgrade.go

```diff
@@ -496,6 +496,14 @@ func ASIGenesisUpgradeASISupply(jsonData map[string]interface{}, networkInfo Net
 	// add the additional coins to the overflow address balance
 	overflowAddressBalanceCoins = overflowAddressBalanceCoins.Add(additionalSupplyCoin)
 
+	// add the new supply mint record to the manifest
+	mintRecord := ASIUpgradeSupplyMint{
```

Comment:
Nit pick - feel free to ignore. 
```suggestion
	supplyRecord := ASIUpgradeSupplyMint{
```